testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -159,24 +162,43 @@ Feature: Smoke tests for <client>
     And I click on "Expand All Sections"
     And I should see a "Enable and configure Prometheus exporters for managed systems." text
     And I check "node" exporter
+    Then I should see a "Formula saved" text
+
+@skip_for_sle_micro
+  Scenario: Enable Prometheus Apache and Postgres exporter formula on the <client>
+    Given I am on the Systems overview page of this "<client>"
+    When I follow "Formulas" in the content area
+    When I follow "Prometheus Exporters" in the content area
+    And I click on "Expand All Sections"
+    And I should see a "Enable and configure Prometheus exporters for managed systems." text
     And I check "apache" exporter
     And I check "postgres" exporter
     And I click on "Save"
     Then I should see a "Formula saved" text
-    # Apply highstate for Prometheus exporters
+
+@skip_for_sle_micro_ssh_minion
+  Scenario: Apply highstate for the Prometheus exporters on the <client>
+    Given I am on the Systems overview page of this "<client>"
     When I follow "States" in the content area
     And I click on "Apply Highstate"
     Then I should see a "Applying the highstate has been scheduled." text
     And I wait until event "Apply highstate scheduled by admin" is completed
-    # Visit monitoring endpoints on the minion
+    And I start "prometheus-node_exporter.service" service on "<client>"
+    And I enable "prometheus-node_exporter.service" service on "<client>"
```

Comment:
does it need to be explicitly started and enabled? shouldn't the highstate be taking care of that?

Reply:
Yes, it must be handled manually because of https://bugzilla.suse.com/show_bug.cgi?id=1209374.

Reply:
this sounds like a step only for sle micro then - can you separate it into a different scenario just for SLE Micro? The other systems should work without this with just the Highstate

Reply:
Yes, that is SLE Micro specific. In BV that is sadly not that easy with the way the test suite report works. As you can see, we nearly always use tags like `@skip_for_*`. @Bischoff can tell you more about that.

Reply:
what if instead of "client" we substitute it for SLE Micro, in these 2 steps? :thinking: that way, it won't apply to the other clients

Reply:
This would work, but it would still show up in the test suite report for other non SLE Micro clients IIRC. I sadly have no example or screenshot to demonstrate that.

Reply:
it's fine for me if it shows up, as long as it doesn't execute. I just want to be sure the other clients are being tested appropriately, where the highstate should be enough to take care of the services being started and enabled. you can leave a comment before those 2 steps specifying the bug you linked with a TODO or WORKAROUND so we know to take it out in the future

Reply:
that would work for me :thinking: @Bischoff @ktsamis what do you think?

Reply:
https://github.com/uyuni-project/uyuni/pull/7525/commits/2898557e33db993703223184ee9d82336eb26431

Reply:
Yes this is the better, cleaner way to handle this and I especially like the workaround note. The separation of this with the tag is the cleaner way to do this so we don't hide any future bugs accidentally. Thanks for this :)